tests/e2e/tuning_config.go

```diff
@@ -0,0 +1,96 @@
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+)
+
+var _ = Describe("Tuning Config", func() {
+	var (
+		tcService *exec.TuningConfigService
+		tcArgs    *exec.TuningConfigArgs
+	)
+
+	verifyTuningConfigSpec := func(spec interface{}, specVmDirtyRatio, specPriority int) {
+		tcSpec := spec.(map[string]interface{})
+		tcProfileSpec := (tcSpec["profile"].([]interface{}))[0].(map[string]interface{})
+		tcRecommendSpec := (tcSpec["recommend"].([]interface{}))[0].(map[string]interface{})
+		Expect(tcProfileSpec["data"]).To(ContainSubstring(fmt.Sprintf("vm.dirty_ratio=\"%d\"", specVmDirtyRatio)))
+		Expect(tcRecommendSpec["priority"]).To(BeEquivalentTo(specPriority))
+	}
+
+	BeforeEach(func() {
+		tcService = exec.NewTuningConfigService(constants.TuningConfigDir)
+	})
+
+	AfterEach(func() {
+		tcService.Destroy()
+	})
+
+	It("can create/edit/delete - [id:72521]",
+		ci.Day2, ci.High, ci.FeatureTuningConfig, func() {
+
+			tcCount := 1
+			specVMDirtyRatios := []int{65}
+			specPriorities := []int{20}
+			By("Create one tuning config")
+			tcArgs = &exec.TuningConfigArgs{
+				Cluster:           clusterID,
+				NamePrefix:        "tc",
```

Comment:
How about has case ID in the Name prefix? So if any resource leak we can know which case caused that.